### ISSUE TITLE
Problem with a DK class hall perk

### ISSUE THREAD

**Issue by Javox9** [Kkthnx/KkthnxUI_Legion#119]
DKs have a class hall perk that allows you to complete a WQ for free every 18h. When I try to complete it, game says that Kkthnx has been blocked by an action only available for Blizzard´s UI.
![wowscrnshot_122616_160849](https://cloud.githubusercontent.com/assets/24768817/21483628/ac183284-cb85-11e6-9726-e2973860bff5.jpg)

**Comment by Kkthnx** [comment]
Here is a walkthrough on how taints work.

To clarify even more. If another addon switches the :Show() method of a secure frame out with something else, then that frame is tainted. However, the taint report will only name the addon that called the tainted :Show() method.

So while a bug report says "KkthnxUI broke a door", a taint report says "KkthnxUI tried to open a door, but it was broken"

Taint reports are only mentioning the addon that called a secure function that has been tainted, they don't say who caused the taint.

BugGrabber should mention that. Because us developers get tons of wrong bug reports (and blame) as a result of people not understanding that taints aren't bugs.

You could say that taint reports are more like whistleblowers. The addon mentioned revealed the problem but didn't always cause it. :smiley:

So in other words. If you get a taint do not always assume it is my UI causing this.

**Comment by Javox9** [comment]
Ohh I see. I didnt know anything about taints.

But I tried to unload all addons but KkthnxUI and KkthnxUI Config, and it´s still the same. Do you know how can I fix it or why is this taint showing?

I love your UI btw, good job 👍
